ui/client/components/toolbars/status/ProcessInfo.tsx

```diff
@@ -69,15 +57,16 @@ class ProcessInfo extends React.Component<OwnProps & StateProps, State> {
       )
     }
 
+    const size = 32
```

Comment:
Maybe it can be a prop?

Reply:
i've removed this from props, never changed, never used